ruby/forwardfinancing.rb

```diff
@@ -13,60 +14,62 @@
   "lead" => {
     "contacts_attributes" => [
       {
-        "first_name" => "string",
-        "last_name" => "string",
-        "email" => "test@forwardfinancing.com",
-        "title" => "string",
-        "born_on" => "2015-01-01",
-        "home_phone" => "6176781000",
-        "cell_phone" => "6176781000",
-        "ssn" => "234345566",
-        "ownership_date" => "2015-01-01",
+        "first_name" => "#{Faker::Name.first_name}",
+        "last_name" => "#{Faker::Name.last_name}",
+        "email" => "#{Faker::Internet.email}",
+        "title" => "#{Faker::Name.prefix}",
+        "born_on" => "#{Faker::Date.birthday(18, 65).strftime("%Y-%m-%d")}",
+        "home_phone" => "#{Faker::Number.number(10)}",
+        "cell_phone" => "#{Faker::Number.number(10)}",
```

Comment:
Two things, little one first:

1. I generally prefer calling `.to_s` on an object to wrapping them in `"#{s}"`. Template strings are best used for concatenation (adding strings together).

2. Would it be a good idea to change this to support 9 digit numbers as well? The phone number format isn't specified anywhere in our docs, @kyle-ch think we should nail this down? Given that we specify the number as a string (like we must), I could imagine getting phone numbers with dashes, dots, parenthesis, and even extensions. I think the `Faker::PhoneNumber` module has some good test data in this category.

Reply:
1. can you give me example?
2. I am just following format from prior. If we can take in any format that would be nice.

Reply:
1. Resolves offline.
2. While I agree, it would be nice to have this investigated, let's wait and see what @kyle-ch says.